standard/expressions.md

```diff
@@ -4728,7 +4728,7 @@ The run-time processing of a ref conditional expression of the form `b ? ref x :
 - If the `bool` value produced by the step above is `true`, then `x` is evaluated and the resulting variable reference becomes the result of the conditional expression.
 - Otherwise, `y` is evaluated and the resulting variable reference becomes the result of the conditional expression.
 
-> *Note:* When `ref` is present, the expression is a reference variable, and can be assigned using the `= ref` operator. *end note*
+> *Note:* When `ref` is present, the *conditional_expression* returns a variable reference, which can be assigned to a reference variable using the `= ref` operator or passed as a reference/input/output parameter. *end note*
 
 If `ref` is not present, the second and third operands, `x` and `y`, of the `?:` operator control the type of the conditional expression:
```

Comment:
And if `ref` is present?

Something along the lines of the two *variable_reference*s must be of the same type, *T*, and the type of the expression is *reference to variable of type T* maybe?

Reply:
I rearranged the few paragraphs here. The order is now:

1. Type of the result:
    a. For `ref` conditional
    b. note on variable reference
    c. For non-ref conditional
2. Runtime processing
    a. for `ref `conditional
    b. for non-ref conditional.